doc/source/intro.rst

```diff
@@ -84,3 +84,57 @@ provided by a :py:obj:`.Renderer`.  For example:
 
     Screen.wrapper(demo)
 
+You can also use the :py:obj:`.Scene.session` class in two different ways.
```

Comment:
Thanks for thinking about documentation!  Many people forget to do this...   That said, I think that this text belongs inside "creating a screen" inside io.rst.  Could you please move it and use the simpler code sample already there (as this makes it clearer what you've changed in both cases)?

Oh and please update the references to Screen.session if you agree that we should rename the class.

Reply:
That makes sense, I'll update it!